Comparing source compatibility of /Users/oovever/Documents/projcet/Github/netty/codec-xml/target/netty-codec-xml-4.1.80.Final-SNAPSHOT.jar against /Users/oovever/.m2/repository/io/netty/netty-codec-xml/4.1.79.Final/netty-codec-xml-4.1.79.Final.jar
WARNING: You have ignored certain classes, i.e. superclasses and interfaces that could not be found on the classpath are ignored. Hence changes caused by these superclasses and interfaces are not reflected in the output.
===  UNCHANGED CLASS: PUBLIC io.netty.handler.codec.xml.XmlAttribute  (not serializable)
	***! CLASS FILE FORMAT VERSION: 51.0 <- 50.0
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED CONSTRUCTOR: PUBLIC XmlAttribute(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)
	===  UNCHANGED METHOD: PUBLIC boolean equals(java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC int hashCode()
	===  UNCHANGED METHOD: PUBLIC java.lang.String name()
	===  UNCHANGED METHOD: PUBLIC java.lang.String namespace()
	===  UNCHANGED METHOD: PUBLIC java.lang.String prefix()
	===  UNCHANGED METHOD: PUBLIC java.lang.String toString()
	===  UNCHANGED METHOD: PUBLIC java.lang.String type()
	===  UNCHANGED METHOD: PUBLIC java.lang.String value()
===  UNCHANGED CLASS: PUBLIC io.netty.handler.codec.xml.XmlCdata  (not serializable)
	***! CLASS FILE FORMAT VERSION: 51.0 <- 50.0
	===  UNCHANGED SUPERCLASS: io.netty.handler.codec.xml.XmlContent (<- io.netty.handler.codec.xml.XmlContent)
	===  UNCHANGED CONSTRUCTOR: PUBLIC XmlCdata(java.lang.String)
===  UNCHANGED CLASS: PUBLIC io.netty.handler.codec.xml.XmlCharacters  (not serializable)
	***! CLASS FILE FORMAT VERSION: 51.0 <- 50.0
	===  UNCHANGED SUPERCLASS: io.netty.handler.codec.xml.XmlContent (<- io.netty.handler.codec.xml.XmlContent)
	===  UNCHANGED CONSTRUCTOR: PUBLIC XmlCharacters(java.lang.String)
===  UNCHANGED CLASS: PUBLIC io.netty.handler.codec.xml.XmlComment  (not serializable)
	***! CLASS FILE FORMAT VERSION: 51.0 <- 50.0
	===  UNCHANGED SUPERCLASS: io.netty.handler.codec.xml.XmlContent (<- io.netty.handler.codec.xml.XmlContent)
	===  UNCHANGED CONSTRUCTOR: PUBLIC XmlComment(java.lang.String)
===  UNCHANGED CLASS: PUBLIC ABSTRACT io.netty.handler.codec.xml.XmlContent  (not serializable)
	***! CLASS FILE FORMAT VERSION: 51.0 <- 50.0
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED CONSTRUCTOR: PROTECTED XmlContent(java.lang.String)
	===  UNCHANGED METHOD: PUBLIC java.lang.String data()
	===  UNCHANGED METHOD: PUBLIC boolean equals(java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC int hashCode()
	===  UNCHANGED METHOD: PUBLIC java.lang.String toString()
===  UNCHANGED CLASS: PUBLIC io.netty.handler.codec.xml.XmlDecoder  (not serializable)
	***! CLASS FILE FORMAT VERSION: 51.0 <- 50.0
	===  UNCHANGED INTERFACE: io.netty.channel.ChannelInboundHandler
	===  UNCHANGED INTERFACE: io.netty.channel.ChannelHandler
	===  UNCHANGED SUPERCLASS: io.netty.handler.codec.ByteToMessageDecoder (<- io.netty.handler.codec.ByteToMessageDecoder)
	===  UNCHANGED CONSTRUCTOR: PUBLIC XmlDecoder()
	===  UNCHANGED METHOD: PROTECTED void decode(io.netty.channel.ChannelHandlerContext, io.netty.buffer.ByteBuf, java.util.List)
		===  UNCHANGED EXCEPTION: java.lang.Exception
===  UNCHANGED CLASS: PUBLIC FINAL io.netty.handler.codec.xml.XmlDocumentEnd  (not serializable)
	***! CLASS FILE FORMAT VERSION: 51.0 <- 50.0
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL io.netty.handler.codec.xml.XmlDocumentEnd INSTANCE
===  UNCHANGED CLASS: PUBLIC io.netty.handler.codec.xml.XmlDocumentStart  (not serializable)
	***! CLASS FILE FORMAT VERSION: 51.0 <- 50.0
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED CONSTRUCTOR: PUBLIC XmlDocumentStart(java.lang.String, java.lang.String, boolean, java.lang.String)
	===  UNCHANGED METHOD: PUBLIC java.lang.String encoding()
	===  UNCHANGED METHOD: PUBLIC java.lang.String encodingScheme()
	===  UNCHANGED METHOD: PUBLIC boolean equals(java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC int hashCode()
	===  UNCHANGED METHOD: PUBLIC boolean standalone()
	===  UNCHANGED METHOD: PUBLIC java.lang.String toString()
	===  UNCHANGED METHOD: PUBLIC java.lang.String version()
===  UNCHANGED CLASS: PUBLIC io.netty.handler.codec.xml.XmlDTD  (not serializable)
	***! CLASS FILE FORMAT VERSION: 51.0 <- 50.0
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED CONSTRUCTOR: PUBLIC XmlDTD(java.lang.String)
	===  UNCHANGED METHOD: PUBLIC boolean equals(java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC int hashCode()
	===  UNCHANGED METHOD: PUBLIC java.lang.String text()
	===  UNCHANGED METHOD: PUBLIC java.lang.String toString()
===  UNCHANGED CLASS: PUBLIC ABSTRACT io.netty.handler.codec.xml.XmlElement  (not serializable)
	***! CLASS FILE FORMAT VERSION: 51.0 <- 50.0
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED CONSTRUCTOR: PROTECTED XmlElement(java.lang.String, java.lang.String, java.lang.String)
	===  UNCHANGED METHOD: PUBLIC boolean equals(java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC int hashCode()
	===  UNCHANGED METHOD: PUBLIC java.lang.String name()
	===  UNCHANGED METHOD: PUBLIC java.lang.String namespace()
	===  UNCHANGED METHOD: PUBLIC java.util.List namespaces()
	===  UNCHANGED METHOD: PUBLIC java.lang.String prefix()
	===  UNCHANGED METHOD: PUBLIC java.lang.String toString()
===  UNCHANGED CLASS: PUBLIC io.netty.handler.codec.xml.XmlElementEnd  (not serializable)
	***! CLASS FILE FORMAT VERSION: 51.0 <- 50.0
	===  UNCHANGED SUPERCLASS: io.netty.handler.codec.xml.XmlElement (<- io.netty.handler.codec.xml.XmlElement)
	===  UNCHANGED CONSTRUCTOR: PUBLIC XmlElementEnd(java.lang.String, java.lang.String, java.lang.String)
	===  UNCHANGED METHOD: PUBLIC java.lang.String toString()
===  UNCHANGED CLASS: PUBLIC io.netty.handler.codec.xml.XmlElementStart  (not serializable)
	***! CLASS FILE FORMAT VERSION: 51.0 <- 50.0
	===  UNCHANGED SUPERCLASS: io.netty.handler.codec.xml.XmlElement (<- io.netty.handler.codec.xml.XmlElement)
	===  UNCHANGED CONSTRUCTOR: PUBLIC XmlElementStart(java.lang.String, java.lang.String, java.lang.String)
	===  UNCHANGED METHOD: PUBLIC java.util.List attributes()
	===  UNCHANGED METHOD: PUBLIC boolean equals(java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC int hashCode()
	===  UNCHANGED METHOD: PUBLIC java.lang.String toString()
===  UNCHANGED CLASS: PUBLIC io.netty.handler.codec.xml.XmlEntityReference  (not serializable)
	***! CLASS FILE FORMAT VERSION: 51.0 <- 50.0
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED CONSTRUCTOR: PUBLIC XmlEntityReference(java.lang.String, java.lang.String)
	===  UNCHANGED METHOD: PUBLIC boolean equals(java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC int hashCode()
	===  UNCHANGED METHOD: PUBLIC java.lang.String name()
	===  UNCHANGED METHOD: PUBLIC java.lang.String text()
	===  UNCHANGED METHOD: PUBLIC java.lang.String toString()
===  UNCHANGED CLASS: PUBLIC io.netty.handler.codec.xml.XmlNamespace  (not serializable)
	***! CLASS FILE FORMAT VERSION: 51.0 <- 50.0
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED CONSTRUCTOR: PUBLIC XmlNamespace(java.lang.String, java.lang.String)
	===  UNCHANGED METHOD: PUBLIC boolean equals(java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC int hashCode()
	===  UNCHANGED METHOD: PUBLIC java.lang.String prefix()
	===  UNCHANGED METHOD: PUBLIC java.lang.String toString()
	===  UNCHANGED METHOD: PUBLIC java.lang.String uri()
===  UNCHANGED CLASS: PUBLIC io.netty.handler.codec.xml.XmlProcessingInstruction  (not serializable)
	***! CLASS FILE FORMAT VERSION: 51.0 <- 50.0
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED CONSTRUCTOR: PUBLIC XmlProcessingInstruction(java.lang.String, java.lang.String)
	===  UNCHANGED METHOD: PUBLIC java.lang.String data()
	===  UNCHANGED METHOD: PUBLIC boolean equals(java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC int hashCode()
	===  UNCHANGED METHOD: PUBLIC java.lang.String target()
	===  UNCHANGED METHOD: PUBLIC java.lang.String toString()
===  UNCHANGED CLASS: PUBLIC io.netty.handler.codec.xml.XmlSpace  (not serializable)
	***! CLASS FILE FORMAT VERSION: 51.0 <- 50.0
	===  UNCHANGED SUPERCLASS: io.netty.handler.codec.xml.XmlContent (<- io.netty.handler.codec.xml.XmlContent)
	===  UNCHANGED CONSTRUCTOR: PUBLIC XmlSpace(java.lang.String)
